packages/css/src/page-menu/page-menu.scss

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+@mixin reset-list {
+  box-sizing: border-box;
+  margin-block: 0;
+  padding-inline: 0;
+}
+
+@mixin reset-item {
+  box-sizing: border-box;
+  -webkit-text-size-adjust: 100%;
+}
+
+.amsterdam-page-menu__list {
+  align-items: center;
+  column-gap: var(--amsterdam-page-menu-list-column-gap);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  list-style: none;
+  row-gap: var(--amsterdam-page-menu-list-row-gap);
+
+  @include reset-list;
+}
+
+@mixin page-menu-item {
+  color: var(--amsterdam-page-menu-item-color);
+  display: inline-flex;
+  flex-direction: row;
+  font-family: var(--amsterdam-page-menu-item-font-family);
+  font-size: var(--amsterdam-page-menu-item-narrow-font-size);
+  font-weight: var(--amsterdam-page-menu-item-narrow-font-weight);
+  gap: var(--amsterdam-page-menu-item-gap);
+  line-height: var(--amsterdam-page-menu-item-line-height);
+  text-align: center;
+  text-decoration: var(--amsterdam-page-menu-item-text-decoration);
+  touch-action: manipulation;
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-page-menu-item-wide-font-size);
+  }
+}
+
+.amsterdam-page-menu__link {
+  @include page-menu-item;
+  @include reset-item;
+}
+
+.amsterdam-page-menu__button {
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  margin-block: 0;
+  padding-inline: 0;
```

Comment:
All of these, except for the `pointer`, could be in a `reset-button` mixin?

Reply:
Sure, but a mixin is for re-usability and it wouldn't be needed elsewhere.

Reply:
Also I personally feel that the [css was cleaner](https://github.com/Amsterdam/design-system/blob/32ccadd4c9d29844b09eaf0f64f3a4169be08557/packages/css/src/page-menu/page-menu.scss) before trying to resolve feedback, the rules applied to both the link and button were not a cause for concern.

Reply:
We currently reuse none of our mixins.

We use reset mixins to separate reset styles from design choices, to prevent having multiple `/* UA override */` comments in a declaration block. See them as the original ‘reset.css’ but then a single one per component, as we avoid tight coupling of all components with a shared stylesheet.

Reply:
I’ve applied this.